utils.js

```diff
@@ -375,7 +375,8 @@ function formatDeltaMessage(m){
     messageID: md.messageId,
     attachments: (m.delta.attachments || []).map(v => _formatAttachment(v)),
     timestamp: md.timestamp,
-    isGroup: !!md.threadKey.threadFbId
+    isGroup: !!md.threadKey.threadFbId,
+    reactions: originalMessage.reactions ? originalMessage.reactions : []
```

Comment:
Am I going crazy or is originalMessage not defined here?  Can you make sure to test this change?  deltaMessage can have slightly different behaviour than the other one (unless you're not getting delta messages?).